bundles/af-core/src/test/resources/form/panelcontainer/exporter-panelcontainer.json

```diff
@@ -6,6 +6,7 @@
   "name":"abc",
   "visible":true,
   "enabled":true,
+  "readOnly": false,
```

Comment:
this should be done in af2-docs first.

Reply:
have a ticket for the same.
https://jira.corp.adobe.com/browse/CQ-4351366
This is a test collateral where we check the emitted json.